lib/config.js

```diff
@@ -1,20 +1,16 @@
 var config = {}
 
-config.collectInterval = process.env.TRACE_COLLECT_INTERVAL || 2 * 60 * 1000
+config.collectInterval = 2 * 60 * 1000
```

Comment:
environment variables have been separated from default config, because the config file's precedence is between the two